modules/AllBooks.js

```diff
@@ -0,0 +1,47 @@
+import { DateTime } from 'luxon';
+import addBook from './addBook.js';
+import AllBooks from '../index.js';
+
+const mybooks = new AllBooks();
+
+const updateTime = () => {
+  const currentDate = DateTime.local();
+  const dateTimeString = currentDate.toLocaleString(DateTime.DATETIME_SHORT);
+  document.getElementById('time').textContent = dateTimeString;
+};
```

Comment:
- [ ] I notice that you have defined a function `updateTime` in your `index.js` file. Yet, you are redefining it here, which is not DRY. Kindly fix that ❤️